service/controller/resource/values/create.go

```diff
@@ -0,0 +1,67 @@
+package values
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/giantswarm/config-controller/pkg/generator/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	app, err := ToAppCR(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	configVersion, ok := app.GetAnnotations()[key.ConfigVersion]
```

Comment:
Do we need to do something to app-operator now?

Reply:
Yes. `app-operator` creates values and secret now. We need to prevent this from happening.

Reply:
@vol is already on it: https://github.com/giantswarm/giantswarm/issues/14604